x-pack/plugins/maps/server/mvt/get_tile.ts

```diff
@@ -91,6 +100,79 @@ export async function getGridTile({
       )
       .toPromise();
     const features: Feature[] = convertRegularRespToGeoJson(response.rawResponse, requestType);
+
+    if (features.length) {
+      const bounds = formatEnvelopeAsPolygon({
+        maxLat: tileBounds.top_left.lat,
+        minLat: tileBounds.bottom_right.lat,
+        maxLon: tileBounds.bottom_right.lon,
+        minLon: tileBounds.top_left.lon,
+      });
+
+      const fieldNames = new Set<string>();
+      features.forEach((feature) => {
+        for (const key in feature.properties) {
+          if (feature.properties.hasOwnProperty(key) && key !== 'key' && key !== 'gridCentroid') {
+            fieldNames.add(key);
+          }
+        }
+      });
+
+      const fieldMeta: FieldMeta = {};
+      fieldNames.forEach((fieldName: string) => {
+        const rangeMeta = pluckRangeFieldMeta(features, fieldName, (rawValue: unknown) => {
+          if (fieldName === COUNT_PROP_NAME) {
+            return parseFloat(rawValue as string);
+          } else if (typeof rawValue === 'number') {
+            return rawValue;
+          } else if (rawValue) {
+            return parseFloat((rawValue as { value: string }).value);
+          } else {
+            return NaN;
+          }
+        });
+
+        const categoryMeta = pluckCategoryFieldMeta(features, fieldName, 20);
```

Comment:
I think 20 should be 30. The largest category palette has 30 items. Also, it should be 30-1 since there is the other category.